### PR TITLE
[NUI] Add DoActionExtension for LottieAnimationView

### PR DESCRIPTION
**Pull request by elishateng**
### Description of Change ###

1. Add DoActionExtension for LottieAnimationView

### API Changes ###

 - ACR: None.

### CODE REVIEW

**Review by zg2nets — APPROVED** [review]
lgtm

File: src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs [hunk 1]
```diff
@@ -821,6 +841,33 @@ public enum LoopingModeType
             /// <since_tizen> 7 </since_tizen>
             AutoReverse
         }
+
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum VectorProperty
+        {
+            FillColor,         ///< Fill color of the object, Type Property::VECTOR3
```
Comment (everLEEst) [inline src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs:849]: please check and resolve all warning messages.

File: src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs [hunk 2]
```diff
@@ -722,6 +728,20 @@ public void SetMinMaxFrameByMarker(string marker1, string marker2 = null)
             Tizen.Log.Error("NUI", $"[ERR] fail to get play range from dali! case#2");
             return new Tuple<int, int>(-1, -1);
         }
+
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
```
Comment (everLEEst) [inline src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs:732]: please add summary.

File: src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs [hunk 3]
```diff
@@ -722,6 +728,20 @@ public void SetMinMaxFrameByMarker(string marker1, string marker2 = null)
...
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [UnmanagedFunctionPointer(CallingConvention.Cdecl)]
+        public delegate PropertyValue DynamicPropertyCallbackType(int id, int returnType, uint frameNumber);
+
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
```
Comment (everLEEst) [inline src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs:737]: please add summary.

File: src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs [hunk 4]
```diff
@@ -821,6 +841,33 @@ public enum LoopingModeType
             /// <since_tizen> 7 </since_tizen>
             AutoReverse
         }
+
```
Comment (everLEEst) [inline src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs:844]: same here.

File: src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs [hunk 5]
```diff
@@ -821,6 +841,33 @@ public enum LoopingModeType
...
+            StrokeColor,       ///< Stroke color of the object, Type Property::VECTOR3
+            StrokeOpacity,      ///< Stroke opacity of the object, Type Property::FLOAT
+            StrokeWidth,       ///< Stroke width of the object, Type Property::FLOAT
+            TransformAnchor,   ///< Transform anchor of the Layer and Group object, Type Property::VECTOR2
+            TransformPosition, ///< Transform position of the Layer and Group object, Type Property::VECTOR2
+            TransformScale,    ///< Transform scale of the Layer and Group object, Type Property::VECTOR2 [0..100]
+            TransformRotation, ///< Transform rotation of the Layer and Group object, Type Property::FLOAT [0..360] in degrees
+            TransformOpacity   ///< Transform opacity of the Layer and Group object, Type Property::FLOAT
+        };
+
```
Comment (everLEEst) [inline src/Tizen.NUI/src/public/BaseComponents/LottieAnimationView.cs:860]: here also.

**Review by everLEEst — CHANGES_REQUESTED** [review]

**Comment by dongsug-song** [comment]
@elishateng 
Hello, 
Thank you for your work.
but this functionality has been added by the following patches, so the PR is not needed currently.

https://github.com/Samsung/TizenFX/pull/4467
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/279320/

so I am just closing this PR.
I am sorry to close your PR without your agreeing, but please understand the situation and not be confused.
thank you.

**Pull request closed without merging**